hydra-cluster/src/CardanoClient.hs

```diff
@@ -69,64 +57,6 @@ buildRaw ins outs fee =
       TxMintValueNone
       TxScriptValidityNone
 
-build ::
-  NetworkId ->
-  FilePath ->
-  Address ShelleyAddr ->
-  -- | Unspent transaction outputs to spend.
-  UTxO ->
-  -- | Collateral inputs.
-  [TxIn] ->
-  -- | Outputs to create.
-  [TxOut CtxTx] ->
-  IO (Either TxBodyErrorAutoBalance TxBody)
-build networkId socket changeAddress utxoToSpend collateral outs = do
-  pparams <- queryProtocolParameters networkId socket QueryTip
-  systemStart <- querySystemStart networkId socket QueryTip
-  eraHistory <- queryEraHistory networkId socket QueryTip
-  stakePools <- queryStakePools networkId socket QueryTip
-  pure $
-    second balancedTxBody $
-      makeTransactionBodyAutoBalance
-        BabbageEraInCardanoMode
-        systemStart
-        eraHistory
-        pparams
-        stakePools
-        (UTxO.toApi utxoToSpend)
-        (bodyContent pparams)
-        (ShelleyAddressInEra changeAddress)
-        noOverrideWitness
- where
-  bodyContent pparams =
-    TxBodyContent
-      (map mkWitness $ toList $ UTxO.inputSet utxoToSpend)
-      (TxInsCollateral collateral)
-      TxInsReferenceNone
-      outs
-      TxTotalCollateralNone
-      TxReturnCollateralNone
-      dummyFee
-      (TxValidityNoLowerBound, TxValidityNoUpperBound)
-      (TxMetadataInEra (TxMetadata noMetadataMap))
-      (TxAuxScripts [])
-      (TxExtraKeyWitnesses [])
-      (BuildTxWith $ Just pparams)
-      (TxWithdrawals WithdrawalsInBabbageEra [])
-      (TxCertificates CertificatesInBabbageEra [] (BuildTxWith noStakeCredentialWitnesses))
-      TxUpdateProposalNone
-      (TxMintValue noMintedValue (BuildTxWith noPolicyIdToWitnessMap))
-      TxScriptValidityNone
-  noMintedValue = mempty
-  noPolicyIdToWitnessMap = mempty
-  noMetadataMap = mempty
-  noStakeCredentialWitnesses = mempty
-  noOverrideWitness = Nothing
-  dummyFee = TxFeeExplicit (Lovelace 0)
-
-  mkWitness :: TxIn -> (TxIn, BuildTxWith BuildTx (Witness WitCtxTxIn))
-  mkWitness txIn = (txIn, BuildTxWith $ KeyWitness KeyWitnessForSpending)
```

Comment:
Moved to `hydra-node` and renamed as `buildTransaction`

Cf discussion in https://github.com/input-output-hk/hydra-poc/discussions/453